natives.json

```diff
@@ -6169,34 +6169,34 @@
 			"build": "323"
 		},
 		"0x661B5C8654ADD825": {
-			"name": "_0x661B5C8654ADD825",
+			"name": "_SET_RADAR_ROT_TO_CAMERA_ROT",

```

Comment:
I'm unsure how I could change this to match _SET_CAM_ because nothing about the camera is being set in this native. The radar's rotation is being set to the camera's rotation, not the other way around. If you have any suggestions I would be happy to change the name.

Reply:
Currently the only name I can think of is something like `_SET_CAM_CONTROLS_RADAR_ROTATION`.